src/Native/System.Native/pal_networking.h

```diff
@@ -117,6 +117,23 @@ struct LingerOption
     int32_t Seconds; // Number of seconds to linger for
 };
 
+struct IOVector
+{
+    uint8_t* Base;
+    uintptr_t Count;
+};
+
+struct MessageHeader
+{
+    uint8_t* SocketAddress;
```

Comment:
Please do not use `unsigned char`: the `char` type is not guaranteed to be exactly 8 bits in length. Please use `uint8_t` (and other explicitly-sized integral types as appropriate).